### PR TITLE
Add support for __OW_IGNORE_CERT env var

### LINKED ISSUE #101: please add ignore_certs env var
When running test code for openwhisk actions it is often necessary to ignore certificates since it is running openwhisk in travis. It looks like the other constructor parameters are already handled this way.

### PR DESCRIPTION
**Pull request by ScottChapman**
fixes #101

### CODE REVIEW

**Comment by rabbah** [comment]
I wouldn’t say this fixes #101 since this new value wont be set in any openwhisk deployment.

**Comment by ScottChapman** [comment]
@rabbah - I think that aspect of it we should track as a separate issue. Should I open one in the openwhisk repo? Not really clear the appropriate place for that discussion...

**Comment by rabbah** [comment]
Yes that’s where I’d suggest. I just don’t see the value of this pr without the prerequisite. In fact I think it’ll be confusing: If you use it for local testing of node code (by setting the env var explicitly)  you’ll have a divergence of behavior vs a managed installation and should use the existing ignore cert parameter instead.

**Comment by ScottChapman** [comment]
It at least gives you the option and is consistent with the others. It will help running actions locally that invoke action in a test deployment. But it is certainly less useful without the other...

**Comment by ScottChapman** [comment]
Can someone review please?
fyi - @rabbah

File: lib/client.js [hunk 1]
```diff
@@ -62,7 +62,10 @@ class Client {
 
   parse_options (options) {
     const api_key = options.api_key || process.env['__OW_API_KEY']
-    const ignore_certs = options.ignore_certs || false
+    const ignore_certs = options.ignore_certs ||
+      (process.env['__OW_IGNORE_CERTS']
```
Comment (ert78gb) [inline lib/client.js]: You cloud write it shorter 
`(process.env['__OW_IGNORE_CERTS'] && process.env['__OW_IGNORE_CERTS'].toLowerCase() === "true") ` not need the ternary operator

File: test/unit/client.test.js [hunk 2]
```diff
@@ -32,15 +32,18 @@ test('should use uuid from auth key as space guid if apigw_token present', t =>
 test('should use environment parameters for options if not set explicitly.', t => {
   process.env['__OW_API_KEY'] = 'some_user:some_pass'
   process.env['__OW_API_HOST'] = 'mywhiskhost'
+  process.env['__OW_IGNORE_CERTS'] = true
```
Comment (ert78gb) [inline test/unit/client.test.js]: in node.js the environment variables are always strings, pls use string value instead if boolean

File: lib/client.js (hunk 1 above)
Comment (ScottChapman) [reply lib/client.js]: That ended up defaulting to `undefined`

**Comment by ScottChapman** [comment]
bump

**Comment by jthomas** [comment]
@ScottChapman Hey Scott, I'd like to merge this but there's now conflicts in the files? Can you re-base and re-push?

Also, can you remove the `package.json` re-formatting?

**Comment by ScottChapman** [comment]
@jthomas - Look better?

**Pull request merged**